packages/react-native-codegen/src/parsers/parsers-commons.js

```diff
@@ -106,15 +106,9 @@ function assertGenericTypeAnnotationHasExactlyOneTypeParameter(
 function isObjectProperty(property: $FlowFixMe, language: ParserType): boolean {
```

Comment:
This could be moved to the `Parser`interface and properly split between `FlowParser` and `TypeScriptParser`. What do you think?

Reply:
Fixed. Calling `isObjectProperty` here is unnecessary, I deleted it. But functions are still moved into `Parser`.